---
layout: papers
title: The variable codons of H3 influenza A virus haemagglutinin genes
authors: ['Gibbs MJ', 'Wayper P', 'Fourment M', 'Wood JT', 'Ohshima K', 'Armstrong JS', 'Gibbs AJ']
year: 2007
journal: Arch Virol
journalref: 152:11-24
doi: 10.1007/s00705-006-0834-8
---

# Abstract

We have analyzed several sets of well-studied haemagglutinin (HA) gene sequences of H3 subtype influenza A viruses to identify codons that are unusually variable, using a simple pairwise sliding window method, DnDscanning. For two of the sets there were results of detailed phylogenetic modeling studies of selection already published. A third set had been the subject of an antigen mapping study, the results of which provide a completely independent benchmark of selected changes in H3 HA genes. Our analyses show that the codons with greatest DnDscan scores (i.e. the most variable) were mostly those reported in the published studies as being positively selected; indeed the DnDscan results matched the antigenic mapping results more closely than did those of the phylogenetic modeling methods. These results suggest that codons under selection can be found even when, as with some sets of virus sequences, a phylogeny is uncertain or cannot be obtained because, for example, the sequences are recombinants, or when selection is not necessarily linked with phylogeny, as in host-switching events. The program DnDscan is available at http://biojanus.anu.edu.au