---
layout: papers
title: Accumulating variation at conserved sites in potyvirus genomes is driven by species discovery and affects degenerate primer design
authors: ['Zheng L', 'Wayper PJ', 'Gibbs AJ', 'Fourment M', 'Rodoni BC', 'Gibbs MJ']
year: 2008
journal: PLoS ONE
journalref: 3:e1586
doi: 10.1371/journal.pone.000158
---

# Abstract

Unknown and foreign viruses can be detected using degenerate primers targeted at conserved sites in the known viral gene sequences. Conserved sites are found by comparing sequences and so the usefulness of a set of primers depends crucially on how well the known sequences represent the target group including unknown sequences.
**METHODOLOGY/PRINCIPAL FINDINGS:**
We developed a method for assessing the apparent stability of consensus sequences at sites over time using deposition dates from Genbank. We tested the method using 17 conserved sites in potyvirus genomes. The accumulation of knowledge of sequence variants over 20 years caused 'consensus decay' of the sites. Rates of decay were rapid at all sites but varied widely and as a result, the ranking of the most conserved sites changed. The discovery and reporting of sequences from previously unknown and distinct species, rather than from strains of known species, dominated the decay, indicating it was largely a sampling effect related to the progressive discovery of species, and recent virus mutation was probably only a minor contributing factor.
**CONCLUSION/SIGNIFICANCE:**
We showed that in the past, the sampling bias has misled the choice of the most conserved target sites for genus specific degenerate primers. The history of sequence discoveries indicates primer designs should be updated regularly and provides an additional dimension for improving the design of degenerate primers.