---
layout: papers
title: Evolutionary dynamics of the N1 neuraminidases of the main lineages of influenza A viruses
authors: ['Fourment M', 'Wood JT', 'Gibbs AJ', 'Gibbs MJ']
year: 2010
journal: Mol Phylogenet Evol
journalref: 56:526-35
doi: 10.1016/j.ympev.2010.04.039
---

# Abstract

Influenza A virus infects a wide range of hosts including birds, humans, pigs, horses, and other mammals. Because hosts differ in immune system structure and demography, it is therefore expected that host populations leave different imprints on the viral genome. In this study, we investigated the evolutionary trajectory of the main lineages of N1 type neuraminidase (NA) gene sequences of influenza A viruses by estimating their evolutionary rates and the selection pressures exerted upon them. We also estimated the time of emergence of these lineages. The Eurasian (avian-like) and North American (classical) swine lineages, the human (seasonal) and avian H5N1 lineages, and a long persisting avian lineage were studied and compared. Nucleotide substitution rates ranged from 1.9x10<sup>-3</sup> to 4.3x10<sup>-3</sup> substitutions per site per year, with the H5N1 lineage estimated to have the greatest rate. The evolutionary rates of the H1N1 human lineage appeared to be slightly greater after it re-emerged in 1977 than before it disappeared in the 1950s. Comparing across the lineages, substitution rates appeared to correlate with the number of positively selected sites and with the degree of asymmetry of the phylogenetic trees. Some lineages had strongly asymmetric trees, implying repeated genotype replacement and narrow genetic diversity. Positively selected sites were identified in all lineages, with the H5N1 lineage having the largest number. A great number of isolates of the H5N1 lineage were sequenced in a short time period and the phylogeny of the lineage was more symmetric. We speculate that the rate and selection estimations made for this lineage could have been influenced by sampling and may not represent the long-term trends.